pymilvus/client/abstract.py

```diff
@@ -44,6 +44,9 @@ def __next__(self):
         self.__index = 0
         raise StopIteration()
 
+    def __str__(self):
+        return str(list(map(str, self.__getitem__(slice(None)))))
```

Comment:
This could be a catastrophe if the top-k/limits is massive, for example, when topk is greater than 10.

Please make a boundary for the returned string. As

```python
return str(list(map(str, self.__getitem__(slice(0, 10)))))
```

Reply:
Requested changes have been made.